lms/static/sass/shared/_footer.scss

```diff
@@ -190,6 +190,10 @@
       }
     }
   }
+
+  @media print {
```

Comment:
I agree that the whole footer shouldn't print, but something out to print. I'm thinking something like a boxed plaintext message that indicates that "This page was printed from {SITE_NAME}. (c) 2015 {PLATFORM_NAME}"